src/components/Search.vue

```diff
@@ -22,21 +20,20 @@ v-layout.searchbar(row, justify-center, align-center)
           v-list-tile-title No results found for query '{{ searchtext }}'
           v-list-tile-sub-title Try an advanced search or refine your query.
       v-list-tile(v-else-if="showMore && searchresults.length > maxQuickResults",
-          @click="$emit('more-results', searchtext)")
+          @click="$emit('more-results', searchtext)",
+          :disabled="!$listeners['more-result']")
```

Comment:
great addition.  You don't see $listeners used much.

Reply:
I learned about this when I see Vuetify component behave differently when based on if there is a listener. So, I think this is safe to do. 🙂